hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializerTest.java

```diff
@@ -161,6 +269,126 @@ public void testReaderReturnsDefaultValues_whenDataIsMissing() {
         assertArrayEquals(new Double[0], innerDTO.nullableDoubles);
     }
 
+    @Test
+    public void testReaderReturnsDefaultValues_whenFieldDoesNotExist() {
```

Comment:
How this is different than `testReaderReturnsDefaultValues_whenDataIsMissing`? (We should update the method name BTW).

Reply:
Not much different(in the new test, data exists but we read with nonexisting field), let me delete this test.